sct_docker.py

```diff
@@ -481,6 +484,7 @@ def generate(distro="debian:7", version="3.1.1", commands=None, name=None,
 		# https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FslInstallation/ShellSetup
 		if False: #os.path.exists("fsl-5.0.11-sources.tar.gz"):
 			frag += "\n" + """
+ENV FSLDIR /home/sct/.local/lib/python3.6/site-packages/fsleyes
```

Comment:
will be overwritten by [this line](ENV FSLDIR /home/sct/fsl)

Reply:
Only if install_fsl is set to True (by defaults it's not): https://github.com/neuropoly/sct_docker/blob/1e5b5261ebf9d2d0c3a26937a0318150abffb097/sct_docker.py#L481

It is hard to read because the terminal command are not subject to the indentation (only the `frag` and `split` commands are). [line 497](https://github.com/neuropoly/sct_docker/pull/51/files#diff-f123f2806960a3fcf71e5e1116184d85R497) in run only if the condition set by the [line 481](https://github.com/neuropoly/sct_docker/blob/1e5b5261ebf9d2d0c3a26937a0318150abffb097/sct_docker.py#L481) and [line 491](https://github.com/neuropoly/sct_docker/blob/1e5b5261ebf9d2d0c3a26937a0318150abffb097/sct_docker.py#L481) are met. if fsl is indeed installed you might need to overwrite it.

Reply:
hum, the script not easy to understand-- first of all, why copying fsl source if `install_fsl=False`? Isn't that a waste of space?

secondly, i would personally move [this line](https://github.com/neuropoly/sct_docker/blob/1e5b5261ebf9d2d0c3a26937a0318150abffb097/sct_docker.py#L487) under a fsleyes-specific section (because it is needed by fsleyes, not FSL)

Reply:
> hum, the script not easy to understand-- first of all, why copying fsl source if install_fsl=False? Isn't that a waste of space?

I am not sure what you mean. [line 486 (`if False`](https://github.com/neuropoly/sct_docker/blob/1e5b5261ebf9d2d0c3a26937a0318150abffb097/sct_docker.py#L485) actually create some unreachable code (which I guess could be deleted then, I made modification before realizing it).  If `install_fsl=false` it would not go inside[ this part](https://github.com/neuropoly/sct_docker/blob/1e5b5261ebf9d2d0c3a26937a0318150abffb097/sct_docker.py#L481-L504) of the code at all.

Reply:
as we discussed you are planning a large refactoring of this script, so i will do a re-review once you're done